internal/start/start.go

```diff
@@ -816,7 +815,7 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			// Stop future runs
 			cancelCtx()
 			// Stop current runs
-			utils.DockerRemoveAll()
```

Comment:
I'm assuming you have tested this, because previously, the network had not been removed there. Are you sure this won't cause issues?

Reply:
Ouch! You are right, I overlooked the suppressed error here https://github.com/supabase/cli/blob/main/internal/start/start.go#L85
(now is handled)

It was still working "thanks" to https://github.com/kandros/supabase-cli/blob/e76bfec2a2efd2239774204a7cb48effd85d623d/internal/start/start.go#L64

This is indeed a bug for me.

fun fact: To prevent a mistake like that is exactly what this refactor was made for in the first place 😅 shame on me